packages/eslint-plugin-next-on-pages/src/rules/no-unsupported-configs.ts

```diff
@@ -93,7 +93,7 @@ const ruleSchema = {
 const rule: Rule.RuleModule = {
 	create: context => {
 		const code = context.sourceCode;
-		const exportedConfigName = context.filename.match(/next\.config\.js$/)
+		const exportedConfigName = context.filename.match(/next\.config\.[mc]?js$/)
 			? getConfigVariableName(code)
```

Comment:
`getConfigVariableName` only works with `module.exports = ...` could you also expand it to take into consideration `export default ...`? (for esm) :pray: 

(PS: we also have a fallback based on the next config type comment but I would prefer it if we could make this more robust and handle the export default as well)

Reply:
Aha, so here is the tricky part. 😅 I will give it a try.

Reply:
I have made a suggestion that seems to work.

Reply:
yeah looks good to me! thanks! 😄